docs/swagger.yaml

```
@@ -591,6 +595,56 @@ paths:
+          description: Bad Request
+        "404":
+          description: Not Found
+        "500":
+          description: Internal Server Error
+      summary: Generate a new police token for a user
+      tags:
+      - user
+  /user/{uuid}/police-token:
+    patch:
+      parameters:
+      - description: User UUID
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 8 but found 6

(indentation)